### DISCUSSION TITLE
Welcome to tinyflux Discussions!

### DISCUSSION THREAD

**Discussion by citrusvanilla** [citrusvanilla/tinyflux#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by drew2323** [comment]
Is tinyflux suitable to store financial timeseries data ?

**Reply by citrusvanilla** [reply]
Sure, why not?  What is the nature of the dataset?  If the data needs to be encrypted and/or protected behind authentication then no, you could choose a better solution than tinyflux but if you are just doing analysis and don't mind have the data exposed as a CSV then it will likely work for you.
